catalog/core/catalog-core-solrcommands/pom.xml

```diff
@@ -125,18 +128,22 @@
                     <groupId>org.eclipse.jetty</groupId>
                     <artifactId>jetty-server</artifactId>
                 </exclusion>
+                <exclusion>
+                    <artifactId>jetty-servlet</artifactId>
+                    <groupId>org.eclipse.jetty</groupId>
+                </exclusion>
             </exclusions>
         </dependency>
         <dependency>
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-server</artifactId>
-            <version>9.3.14.v20161028</version>
+            <version>9.4.10.v20180503</version>
```

Comment:
:question: Same question as above, should we not define these in the parent pom file for DDF

Reply:
Same as above.